packages/modules/order/src/migrations/Migration20240715174100.ts

```diff
@@ -1,6 +1,6 @@
 import { Migration } from "@mikro-orm/migrations"
 
-export class Migration20240715102100 extends Migration {
+export class Migration20240715174100 extends Migration {
```

Comment:
question: should we revert this change?

Reply:
I think the migration will read the file name instead of the Class. In the previous commit I didn't change the name and it was the same as the previous migration at that time.
Should be fine to keep it